### PR TITLE
empty frontend

### PR DESCRIPTION
**Pull request by oboking**

### CODE REVIEW

File: frontend/package.json [hunk 1]
```diff
@@ -0,0 +1,39 @@
+{
+  "name": "frontend",
+  "version": "0.1.0",
+  "private": true,
```
Comment (dawidstrom) [inline frontend/package.json:4]: what is this private u talking about

File: frontend/public/index.html [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+    <!--
+      Notice the use of %PUBLIC_URL% in the tags above.
+      It will be replaced with the URL of the `public` folder during the build.
+      Only files inside the `public` folder can be referenced from the HTML.
+
+      Unlike "/favicon.ico" or "favicon.ico", "%PUBLIC_URL%/favicon.ico" will
+      work correctly both with client-side routing and a non-root public URL.
+      Learn how to configure a non-root public URL by running `npm run build`.
+    -->
+    <title>Jackbox</title>
```
Comment (dawidstrom) [inline frontend/public/index.html:27]: hmm

File: frontend/src/App.tsx [hunk 3]
```diff
@@ -0,0 +1,12 @@
+import React from 'react';
+import './App.css';
+
+const App = () => {
+  return (
+    <div data-testid="app-wrapper" className="App">
+      davids mamma
```
Comment (dawidstrom) [inline frontend/src/App.tsx:7]: Reeeee

File: frontend/package.json (hunk 1 above)
Comment (oboking) [reply frontend/package.json:4]: omg u stupido

**Review by albinvass — APPROVED** [review]

**Pull request merged**